packages/react-core/src/filters/tileFeaturesGeometries.js

```diff
@@ -4,36 +4,84 @@ import booleanWithin from '@turf/boolean-within';
 import intersect from '@turf/intersect';
 import transformToTileCoords from '../utils/transformToTileCoords';
 import { TILE_FORMATS } from '../types';
+import transformTileCoordsToWGS84 from '../utils/transformTileCoordsToWGS84';
 
 const GEOMETRY_TYPES = Object.freeze({
   Point: 0,
   LineString: 1,
   Polygon: 2
 });
 
+function processTileFeatureProperties({
+  map,
+  data,
+  startIndex,
+  endIndex,
+  type,
+  bbox,
+  tileFormat,
+  uniqueIdProperty,
+  storeGeometry,
+  geometryIntersection = null
+}) {
+  const tileProps = getPropertiesFromTile(data, startIndex);
+  const uniquePropertyValue = getUniquePropertyValue(tileProps, uniqueIdProperty, map);
+
+  if (uniquePropertyValue && !map.has(uniquePropertyValue)) {
```

Comment:
Minor - An early exit might be more readable than nesting `if` blocks, here and below:

```javascript
if (!uniquePropertyValue || map.has(uniquePropertyValue)) {
  return;
}

...

if (geometryIntersection && !intersects(geometry, geometryIntersection)) {
  return;
}
```

Reply:
Yes is more readable, thank you for the suggestion, I've changed it